airbyte-server/src/main/java/io/airbyte/server/RequestLogger.java

```diff
@@ -104,7 +104,7 @@ public void filter(ContainerRequestContext requestContext, ContainerResponseCont
     if (HttpStatus.isClientError(status) || HttpStatus.isServerError(status)) {
       LOGGER.error(logBuilder.toString());
     } else {
-      LOGGER.info(logBuilder.toString());
+      LOGGER.debug(logBuilder.toString());
```

Comment:
I think it makes sense for our API server to log requests by default. We can do things like analyze our own logs for % successful requests / other analytics. It's also a good indicator that things are actually working and when it's receiving requests. This is something that I've actually run into while setting up production instances while debugging connectivity issues.